src/js/common/schemaform/actions.js

```diff
@@ -133,9 +144,14 @@ export function submitForm(formConfig, form) {
       .catch(error => {
         // overly cautious
         const errorMessage = _.get('message', error);
-        const clientError = errorMessage && !errorMessage.startsWith('vets_server_error');
-        captureError(error, clientError);
-        dispatch(setSubmission('status', clientError ? 'clientError' : 'error'));
+        let errorType = 'clientError';
+        if (errorMessage.startsWith('vets_throttled_error')) {
+          errorType = 'throttledError';
+        } else if (errorMessage.startsWith('vets_server_error')) {
+          errorType = 'error';
```

Comment:
Since we're specifying clientError and throttledError, would it make sense to do the same for error (i.e. serverError)?

Reply:
That's probably more clear now, yeah.